src/media/css/detail/more-info.styl

```diff
@@ -25,6 +25,7 @@
     }
     dd {
         display: inline-block;
+        margin-right: 3px;
```

Comment:
I like this but we'll also need to consider RTL. There should be some `+rtl() {}` blocks you can reference.